packages/contracts-bedrock/src/L1/ResourceMetering.sol

```diff
@@ -147,6 +147,16 @@ abstract contract ResourceMetering is Initializable {
         }
     }
 
+    /// @notice Adds an amount of L2 gas consumed to the prev bought gas params. This is meant to be used
+    ///         when L2 system transactions are generated from L1. This function is overflow safe due to
+    ///         the constraint of the max resource limit being much larger than `type(uint64).max`.
```

Comment:
typo in this sentence? `maxResourceLimit` above is a uint32, so it's much smaller than `type(uint64).max`

Reply:
I removed this phrasing, to be honest i am a bit confused as to why i phrased it this way originally. Its a bit of a footgun but we are calling it with a hardcoded value so its safe